exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/internal/logs/OtlpStdoutLogRecordExporter.java

```diff
@@ -53,8 +58,12 @@ public CompletableResultCode export(Collection<LogRecordData> logs) {
     }
 
     if (wrapperJsonObject) {
-      LogsRequestMarshaler request = LogsRequestMarshaler.create(logs);
-      return jsonWriter.write(request);
+      return new LogReusableDataMarshaler(memoryMode) {
```

Comment:
Instantiating a new `LogReusableDataMarshaler` instance for each export request negates the reusable memory mode because a new `Deque<LowAllocationLogsRequestMarshaler>` is established for each export request. This should be moved up to the constructor.

Reply:
great catch